File: URI/1018.c
#include <stdio.h>
 
#define mod(x) ((x >= 0)? x : -1 * x)

//www.urionlinejudge.com.br/judge/en/problems/view/1018
int main() {
    int value = 0;
    int notes[] = {100, 50, 20, 10, 5, 2, 1};

	scanf("%d", &value);
	printf("%d\n", value);

	for (int i = 0; i <  7; i++){
		int aux = value / notes[i];
		printf("%d nota(s) de R$ %d,00\n", aux, notes[i]);
		value = value % notes[i];
	}
 
    return 0;
}

File: URI/golang/1018.go
package main

import "fmt"

// Q1018 https://www.urionlinejudge.com.br/judge/pt/problems/view/1018
func Q1018() {
  var N, amount int
  notes := [7]int{100, 50, 20, 10, 5, 2, 1}

  fmt.Scanf("%d", &N)
  fmt.Printf("%d\n", N)

  for i := 0; i < 7; i++ {
    amount = N / notes[i]
    fmt.Printf("%d nota(s) de R$ %d,00\n", amount, notes[i])
    N = N % notes[i]
  }
}